src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/InProcess/EditorInProcess_Classification.cs

```diff
@@ -128,7 +117,7 @@ public async Task VerifyGetClassificationsAsync(IEnumerable<ClassificationSpan>
             }
         }
 
-        Assert.Equal(expectedArray.Length, actualArray.Length - actualOffset);
+        Assert.Equal(expectedArray.Length, actualArray.Length);
```

Comment:
I'd still argue this line is useless at best. It will either do nothing, or never be reached.